redirects.yaml

```diff
@@ -291,25 +291,23 @@ redirects:
     destination: /media-eme
 
   # Redirects to https://developer.chrome.com/.
-  - source: /browser-flags/
+  - source: /browser-flags
     destination: https://developer.chrome.com/blog/browser-flags/
-  - source: /origin-trials/
```

Comment:
There are two `/origin-trials/` entries in this file. Only one of them will "win."

https://developer.chrome.com/docs/web-platform/origin-trials/ and https://developer.chrome.com/blog/origin-trials/ are the two destinations. The non-`/blog/` one seems more "authoritative," so I'm removing the `/blog/` entry.